### DISCUSSION TITLE
How to make list button open a link

### DISCUSSION THREAD

**Discussion by bilo2000** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#87] [General]
Hi,
is there anyway to make the List Button action to open a URL?
Regards

**Comment by gabrieldwight** [comment]
Hi, I am not sure if the list button supports opening a URL unless I check if the feature was added. I know reply button messages or interactive CTA Url button messages support opening a link. I will push an update if that feature is added for list button messages.

**Comment by bilo2000** [comment]
Hi,

For reply messages, is it avaialbe in your code how to build and send it, and then how to read it

**Reply by gabrieldwight** [reply]
Hi, it is not available. I am not sure if reply list button can open a link unless that feature was added by cloud api

**Comment by bilo2000** [comment]
Hi, i am asking only about the part of you old answer above where you mentioned that we can use "Reply Button", not list buttons

is this also not available?

**Comment by gabrieldwight** [comment]
The reply button is mostly from the template side when created and approved, not the normal one. There is also interactive Call to action buttons for doing the same to open url

**Comment by bilo2000** [comment]
Thanks gabrial
i have fixed the issue as per your help, i used template based messages and it works fine
Regards
